pages/03-core/solidgauge.page.tsx

```diff
@@ -49,9 +59,9 @@ export default function () {
               text: "Usage",
             },
             stops: [
-              [0.1, "#55BF3B"],
-              [0.5, "#DDDF0D"],
-              [0.8, "#DF5353"],
+              [0.1, colors[0]],
```

Comment:
These colors do not improve the situation: those are not designed to have enough colour contrast with labels rendered on top.